bin/autoaggregate

```diff
@@ -53,27 +93,13 @@ def aggregate(config):
             cwd=SRC_DIR,
             stderr=sys.stderr,
             stdout=sys.stdout,
+            preexec_fn=pre_exec,
         )
     finally:
-        # Restore umask, if changed
-        if old_umask is not None:
-            os.umask(old_umask)
         # Chown recursively, if UID or GID are specified
+        # just to make sure everything is owned by odoo:odoo (should run empty)
         if ~UID or ~GID:
-            for root, dirs, files in os.walk(SRC_DIR):
-                for target in dirs + files:
-                    try:
-                        target_path = os.path.join(root, target)
-                        if not os.path.islink(target_path):
-                            # if target_path is a link we would only chown the
-                            # target, as the target should be chowned separately
-                            # we can skip all symlinks and prevent showing
-                            # unclear errors
-                            os.chown(target_path, UID, GID)
-                    except Exception:
-                        logger.debug(
-                            "Error trying to chown on file. Skipping...", exc_info=True
-                        )
+            check_call(["fix_permissions"])
```

Comment:
AFAICS, you're changing the permissions in the system beforehand to avoid chowning later.

If that's the case, what's the point on doing this call? 🤔

Reply:
I am trying to make sure we have the right permissions after autoaggregate to prevent changing it later again. I think i could get rid of it but i was not 100% sure, so i kept fixing the permission here. (currently it only changes some permissions in .git from `r--r-----` to `rw-r-----`)